microcosm_pubsub/handlers/uri_handler.py

```diff
@@ -125,7 +125,8 @@ def get_resource(self, message, uri):
         Passes message context.
 
         """
-        headers = self.sqs_message_context(message)
+        # XXX we only want X-Request headers
```

Comment:
I don't think we need to use `sqs_mexsage_context` here, but we also don't have good tests for this.